src/Package.zig

```diff
@@ -673,8 +701,20 @@ fn fetchAndUnpack(
         std.zig.fmtId(fqn), std.zig.fmtEscapes(build_root),
     });
 
+    {
+        var build_root_dir = try std.fs.openDirAbsolute(build_root, .{});
+        defer build_root_dir.close();
+        build_root_dir.access(build_zig_basename, .{}) catch {
```

Comment:
Please do this with only 1 syscall, and also without using the "absolute" variants of file system functions. Note that `Cache.Directory.join` does not necessarily return an absolute path.

Reply:
Fixed! I also discovered that I forgot to check the error returned from the ```access``` call. We now only continue if the error returned is the ```FileNotFound``` error instead of continuing on every error.